frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -1203,6 +1203,7 @@ A custom wrapper around `fetch` that adds console-specific headers and allows fo
 | `method` | The HTTP method to use. Defaults to GET |
 | `options` | The options to pass to fetch |
 | `timeout` | The timeout in milliseconds |
+| `isFullResponse` | The flag to cotrol whether to return full or partial response. The default is partial. |
```

Comment:
This is too vague. We need to specific about how full or partial response means, ideally with examples. A developer should be able to read this and use the API without needing to experiment or look at the code.

I'm not sure "full" is the right word here. You're not getting a partial response even if this is true. You get the complete response body, just not the headers and status.

Reply:
I'd expect to see doc updates for all of the `k8s*` methods as well.

Reply:
> I'd expect to see doc updates for all of the `k8s*` methods as well.

@spadgett Sure, I can take care of the doc.  Could you please suggest an alternative name instead of `fullResponse`?

Reply:
> > I'd expect to see doc updates for all of the `k8s*` methods as well.
> 
> @spadgett Sure, I can take care of the doc. Could you please suggest an alternative name instead of `fullResponse`?

@spadgett ChatGBT suggests `completeHttpResponse` . Does that look better?